### DISCUSSION TITLE
当半夜设备的win系统的自动化控制运行时，而不是像平时的人去操作，会统计时间吗？

### DISCUSSION THREAD

**Discussion by WinsonMok** [Planshit/Tai#142] [Ideas]
当半夜设备的win系统的自动化控制运行一些像测量软件时，而不是像平时的人去操作，会统计时间吗？
如果不会统计，可否加入这个功能？感谢~

**Comment by noberumotto** [comment]
如果自动化控制的时候鼠标指针不会移动的话会进入睡眠状态，就不会统计时间，如果指针会移动就会被统计。未来的版本会加入一个手动进入睡眠的开关，我想问下自动化控制用的是什么软件或者方案呢

**Reply by WinsonMok** [reply]
我们自动化控制的主要是keysight的测量设备，比如实时示波器等，运行的软件主要如下，晚上自动化运行软件测试电气信号数据时，其实也是在使用的，所以希望把部分也算在统计时间内，感谢~
C:\Program Files\Keysight\lnfiniium\ Apps\EthernetTest Keysight,DigitalTestApps.Framework.AutomatedTestEnqine.exe
C:Program Files\Infiniium\Infiniium.ex

**Reply by noberumotto** [reply]
了解了，谢谢反馈

**Reply by WinsonMok** [reply]
请咨询一下，后面会预计加入对自动化控制的统计时间功能进去吗？，谢谢~

**Reply by noberumotto** [reply]
会加入的

**Reply by WinsonMok** [reply]
hello，冒昧打扰，这个功能预计什么时候会加入呀？感谢~
